src/css/styles/_c-form.scss

```diff
@@ -0,0 +1,185 @@
+    background-size: 187px;
+    border-bottom: solid $col-orange 3px;
+  }
+
+  &__form {
+    position: relative;
+  }
+
+  &__header {
+    padding: 0;
+    padding-bottom: 3rem;
+    font-family: $font-praktika-bold;
+    font-size: 2rem;
+    line-height: 150%;
+    color: $col-white;
+    text-transform: uppercase;
+    letter-spacing: 0.1em;
+  }
+
+  &__close {
+    position: absolute;
+    top: 0;
+    right: 0;
+  }
+
+  &__close-icon {
+    max-width: 28px;
+  }
+
+  &__col-right {
+    padding: 0;
+    padding-left: 1rem;
+  }
+
+  &__col-left {
+    padding: 0;
+    padding-right: 1rem;
+  }
+
+  &__group {
+    display: block;
+    width: 100%;
+    margin-bottom: 1rem;
+  }
+
+  &__group--textarea {
+    height: 100%;
+  }
+
+  &__label {
+    display: block;
+    width: 100%;
+    padding: 1rem 0;
+    font-family: $font-praktika-bold;
+    font-size: 0.6rem;
+    color: $col-orange;
+    text-transform: uppercase;
+    letter-spacing: 0.1em;
+  }
+
+  &__input {
+    width: 100%;
+    padding: 1rem;
+    font-family: $font-praktika-medium;
+    font-size: 1.2rem;
+    color: $col-white;
+    background: $col-bg-gray-dark;
+    border: solid 2px $col-form-border;
+  }
+
+  &__textarea {
+    width: 100%;
+    height: calc(100% - 46px);
+    padding: 1rem;
+    font-family: $font-praktika-medium;
+    font-size: 1rem;
+    color: $col-nav-link;
+    background: $col-bg-gray-dark;
+    border: solid 2px $col-form-border;
+  }
+
+  &__btn {
+    position: relative;
+    width: 100%;
+    padding: 0.6rem;
+    margin-top: 2rem;
+    font-family: $font-praktika-bold;
+    font-size: 0.6rem;
+    color: $col-white;
+    text-transform: uppercase;
+    letter-spacing: 0.1em;
+    border-bottom: solid $col-orange 4px;
+  }
+
+  &__btn:hover {
+    text-decoration: none;
+  }
+
+  &__btn::after {
+    position: absolute;
+    right: 0;
+    bottom: 0;
+    content: url("../img/c-btn--form.png");
+    transform: scale(0.5);
+  }
+
+  &__logo {
+    position: absolute;
+    right: 2rem;
+    bottom: 4rem;
+  }
+
+  &__logo-img {
+    max-width: 187px;
+  }
+
+  @include media-breakpoint-down(lg) {
+    &__header {
+      font-size: 1.6rem;
+    }
+  }
+
+  @include media-breakpoint-down(md) {
+    padding: 2rem;
+    padding-bottom: 6rem;
+
+    &__header {
+      padding-bottom: 1rem;
+      font-size: 1rem;
+    }
+
+    &__label {
+      padding-top: 0;
+    }
+
+    &__textarea {
+      height: calc(100% - 30px);
+    }
+  }
+
+  @include media-breakpoint-down(sm) {
+    &__col-right,
+    &__col-left, {
+      padding: 0;
+    }
+  }
+
+  @include media-breakpoint-down(xs) {
+    padding: 1rem;
+    padding-bottom: 6rem;
+
+    &::after {
+      right: 1rem;
+      bottom: 1rem;
+      width: calc(100% - 2rem);
+    }
+
+    &__group--textarea {
+      height: 350px;
+    }
+
+    &__input {
+      font-size: 1rem;
+    }
+
+    &__textarea {
+      font-size: 0.8rem;
+    }
+  }
+}
```

Comment:
I don't like green border when user fill input that was marked as invalid. Could we go back to grey one?

- [ ] grey instead of green